### DISCUSSION TITLE
Fraudulent Website

### DISCUSSION THREAD

**Discussion by otaviosribeiro** [community/community#75314] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

All my pages related to my account (otaviosribeiro.github.io) are being identified as fraudulent by Google Chrome, I can't find a way to fix it anywhere, I've even tried to reset my repository

**Comment by davixmns** [comment, marked as answer]
This could be due to various reasons, including:

Phishing or Malware: There might be malicious content or code injected into your GitHub Pages that resembles phishing or malware sites. This can trigger Google Chrome's Safe Browsing feature to block your pages.

Suspicious Activity: If your GitHub account or repository has been involved in suspicious activity, such as spamming or hacking attempts, Google might temporarily flag your pages as fraudulent.

Third-Party Links: If your GitHub Pages contain links to external websites that have been flagged as fraudulent, this can also affect the reputation of your own pages.

To resolve this issue, you'll need to identify and address the specific cause of the fraudulent flag. Here are some steps you can take:

Review your GitHub Pages: Carefully examine your GitHub Pages content, including code, scripts, and external links, for any signs of suspicious activity or malicious code.

Check for Malware: Use a reputable malware scanner to scan your GitHub repository for any potential malware or phishing scripts.

Reset your Repository: If you suspect your repository has been compromised, consider resetting it to a clean state from a previous commit.

Contact Google Safe Browsing: If you're confident your pages are legitimate and the flag is a false positive, you can contact Google Safe Browsing to request a review.

Consult GitHub Support: If the issue persists, reach out to GitHub support for further assistance. They can provide more in-depth guidance on identifying and resolving the issue.

Remember, it's crucial to maintain the security and integrity of your GitHub Pages to avoid such issues in the future. Regularly update your code, use secure coding practices, and avoid embedding suspicious content or links.

**Reply by otaviosribeiro** [reply]
Estava vendo e estou achando que tem haver com um site que fiz parecido com o da Netflix para fins de estudos. Acho que o algoritmo identificou como golpe. Porém ja apaguei todos repositórios e nao volta ao normal

**Reply by davixmns** [reply]
pode marcar como respondido mano? pra dar uma força aqui, vlww

**Reply by otaviosribeiro** [reply]
Vou marcar aqui sim mano! mas voce nao sabe como arrumar se for esse caso que estou pensando ne? sou novo na area

**Reply by davixmns** [reply]
talvez o github tenha te marcado por causa desse repositorio. Acho que so criando outra conta ou acionar o suporte
